### DISCUSSION TITLE
Support for Pillow (Python image library)

### DISCUSSION THREAD

**Discussion by sbdchd** [cloudflare/workerd#1941] [Python Packages]
I use this to munge some upload images into background thumbnails as part of a web service

**Comment by zyd16888** [comment]
I need this too

**Comment by ianhco** [comment]
Would appreciate image manipulation library, OpenCV is too heavy, so would like to at least have Pillow, which is already good enough for most lightweight usecases that fits the context of deployment at Cloudflare Workers

**Comment by ryanking13** [comment]
Pillow works in Python workers now, check out the example: https://github.com/cloudflare/python-workers-examples/blob/main/image-gen/src/entry.py
